core/examples/site_themed_example/page1.md

```diff
@@ -0,0 +1,9 @@
+site_example
+=====
+
```

Comment:
I don't think we need this particular example, but it could be reworked to be more suited to introduce new users to the site feature. Something that looks more like site_example2, but more informative. We could link to documentation of the syntax and show how the `page_element` macro is used.